### PR TITLE
User can view all flash cards

### PR DESCRIPTION
**Pull request by heegupark**
view-all-cards

### CODE REVIEW

File: dist/style.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.title-custom {
+  color: gainsboro;
+}
+
+.text-custom {
+  color: white;
+}
```
Comment (thebearingedge) [inline dist/style.css:7]: Bootstrap has a class for this.

File: src/components/app.jsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ class App extends Component {
     super();
     this.state = {
       view: 'view-cards',
-      cards: []
+      cards: JSON.parse(localStorage.getItem('flash-cards')) ? JSON.parse(localStorage.getItem('flash-cards')) : []
```
Comment (thebearingedge) [inline src/components/app.jsx:12]: This is usually shortened to use `||` so you don't have to parse the JSON twice.

```js
JSON.parse(localStorage.getItem('flash-cards')) || []
```

File: dist/index.html [hunk 3]
```diff
@@ -6,11 +6,17 @@
...
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css"
+    integrity="sha256-46qynGAkLSFpVbEBog43gvNhfrOj+BmwXdxFgVK/Kvc=" />
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
   <div id="root"></div>
 
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" crossorigin="anonymous"></script>
```
Comment (thebearingedge) [inline dist/index.html:19]: I don't think you need to load all this extra javascript.

File: src/components/app.jsx (hunk 2 above)
Comment (heegupark) [reply src/components/app.jsx]: Oh got it!

File: dist/index.html (hunk 3 above)
Comment (heegupark) [reply dist/index.html]: Removed popper.js, but rest of them seems to be necessary for making hamburger menu work.

File: dist/style.css (hunk 1 above)
Comment (heegupark) [reply dist/style.css]: Changed!

**Pull request merged**